requirements/dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ pep8
 astroid==1.4.8
 pylint==1.6.4
 tox==2.3.1
+mock
```

Comment:
It wasn't being installed by tox and I saw test failures as a result. Should be easy to reproduce - run 'tox' without that added. I had presumed that travis has some extra deps installed by default; happy to dig further.